src/Actions.js

```diff
@@ -1,12 +1,9 @@
+/* eslint no-use-before-define: ["error", { "variables": false }] */
+
 import PropTypes from 'prop-types';
 import React from 'react';
-import {
-  StyleSheet,
-  Text,
-  TouchableOpacity,
-  View,
-  ViewPropTypes,
-} from 'react-native';
+import { StyleSheet, Text, TouchableOpacity, View, ViewPropTypes } from 'react-native';
+import Colors from './Colors';
```

Comment:
Nit: Should be singular `Color` [see [here](https://stackoverflow.com/a/15756009/763231)]

Reply:
Thanks, I did it...